docs/content/en/main/concepts/syncer.md

```diff
@@ -164,36 +195,41 @@ You can run the syncer in a kind cluster for development.
    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/kubernetes-sigs/kind/main/site/static/examples/kind-with-registry.sh)"
    ```
 
-1. Install `ko`:
-
-    ```sh
-    go install github.com/google/ko@latest
+1. From the kcp root directory:
+    ```bash
+    export KUBECONFIG=.kcp/admin.kubeconfig
     ```
 
-1. Build image and push to the local registry integrated with `kind`:
+1. Create a location workspace and immediately enter it:
 
     ```sh
-    KO_DOCKER_REPO=localhost:5001 ko publish ./cmd/syncer -t <image tag>
+    $ kubectl kcp workspace create location --enter
+    Workspace "location" (type root:organization) created. Waiting for it to be ready...
```

Comment:
What about calling it `my-locations`, to show that it is not a reserved word, like you did below with `my-org` ?